server/accounts.go

```diff
@@ -4181,17 +4201,39 @@ func getMappingFunctionArgs(functionRegEx *regexp.Regexp, token string) []string
 	return nil
 }
 
-// Helper to pull raw place holder indexes and number of partitions. Returns -1 if not a place holder.
-func placeHolderIndex(token string) ([]int, int32, error) {
+// Helper for mapping functions that take a wildcard index and an integer as arguments
+func transformIndexIntArgsHelper(token string, args []string, transformType int16) (int16, []int, int32, string, error) {
+	if len(args) < 2 {
+		return BadTransform, []int{}, -1, "", &mappingDestinationErr{token, ErrorMappingDestinationFunctionNotEnoughArguments}
+	}
+	if len(args) > 2 {
```

Comment:
So basically, if different than 2 it is an error. If we follow the logic then here it should be the "too many" error. I think you have added that one already, so should be available.

Reply:
Doh! Thanks for catching this. I think because when I added the too many args error I used it in one place but forget to update those len(args)>2 to use it as well...